shared/src/main/scala/uk/gov/homeoffice/drt/arrivals/Arrival.scala

```diff
@@ -116,12 +121,31 @@ case class Arrival(Operator: Option[Operator],
     (minutesToDisembark * oneMinuteInMillis).toLong
   }
 
-  val bestPcpPaxEstimate: Int = (ApiPax, ActPax, TranPax, MaxPax) match {
-    case (Some(apiPax), _, _, _) if !FeedSources.contains(LiveFeedSource) => apiPax
-    case (_, Some(actPax), Some(tranPax), _) if (actPax - tranPax) >= 0 => actPax - tranPax
-    case (_, Some(actPax), None, _) => actPax
-    case (Some(apiPax), _, _, _) => apiPax
-    case _ => 0
+  val bestPcpPaxEstimate: TotalPaxSource = (ApiPax, ActPax, TranPax, MaxPax, TotalPax) match {
+    case (_, _, _, _, totalPax) if totalPax.exists(tp => tp.feedSource == ScenarioSimulationSource && tp.pax > 0) =>
+      totalPax.find(tp => tp.feedSource == ScenarioSimulationSource)
+        .getOrElse(TotalPaxSource(0, UnknownFeedSource, None))
```

Comment:
Would be nice to move these 8 repeated `getOrElse` into a single line at the end of the function

Reply:
In fact, this function wants rewriting as the first match is no longer relevant. We can replace the first match with a `totalPax.find` to simplify it a lot. We then don't need the `.exists` calls before each `.find`